controllers/nginx/pkg/cmd/controller/metrics.go

```diff
@@ -40,7 +42,10 @@ func (em exeMatcher) MatchAndName(nacl common.NameAndCmdline) (bool, string) {
 }
 
 func (n *NGINXController) setupMonitor(args []string) {
-	pc, err := newProcessCollector(true, exeMatcher{"nginx", args})
+	var enableVts = true
```

Comment:
@gianrubio please no. Keep in mind we must support to switch the implementation if the `enable-vts-metrics` value is updated in the configmap.